qubole-aws-terraform-deployment/qubole-deployment/modules/hive_metastore/aws_vpc_hive_metastore_vpc.tf

```diff
@@ -0,0 +1,17 @@
+/*
+Creates the Hive Metastore Dedicated VPC.
+
+ This is for the following reason:
+ 1. Isolating the metastore in a dedicated VPC with no DNS support makes it invisible to the internet
+ 2. We can selectively control access to it from external applications like Qubole
+*/
+
+resource "aws_vpc" "hive_metastore_dedicated_vpc" {
+  cidr_block = "11.0.0.0/16"
```

Comment:
Should the CIDR block go into variables?

Reply:
Noted. Moved. Also applied the same changes for the subnet CIDRs